usage-examples/go/atlas-sdk-go/scripts/bluehawk.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+usage() {
+  cat <<EOF
+Usage: $(basename "$0") <command> [flags]
+
+Commands:
+  snip   Extract code examples from Bluehawk snippets
+  copy   Copy sanitized project files
+
+Example (non‐interactive):
+  $(basename "$0") copy --ignore="tests/*.go" --rename='{"old.md":"new.md"}'
+EOF
+  exit 1
+}
+
+# defaults
+CMD=""
+PROJECT=$(git rev-parse --show-toplevel)
+INPUT_DIR="$PROJECT/usage-examples/go/atlas-sdk-go"
+OUTPUT_DIR="$PROJECT/generated-usage-examples/go/atlas-sdk-go/"
+STATE=""
+IGNORE_PATTERNS=(
+  "README.md"
```

Comment:
I am a little curious why we are calling the internal README `README.MD` and then renaming the one that we intend for public consumption. I might suggest the opposite pattern - naming the one for public consumption `README.md`, and instead having an `INTERNAL_README.md` that we add to the ignore patterns. No need for a rename in that case.

Reply:
yeah, that makes way more sense. that also provides "internal_" as an general ignore pattern too